src/it/java/org/jboss/tools/intellij/quarkus/fixtures/dialogs/project/pages/QuarkusNewProjectFinalPage.java

```diff
@@ -14,18 +14,36 @@
 import com.intellij.remoterobot.data.RemoteComponent;
 import com.intellij.remoterobot.fixtures.DefaultXpath;
 import com.intellij.remoterobot.fixtures.FixtureName;
+import com.intellij.remoterobot.fixtures.JTextFieldFixture;
+import com.redhat.devtools.intellij.commonuitest.UITestRunner;
 import com.redhat.devtools.intellij.commonuitest.fixtures.dialogs.project.pages.AbstractNewProjectFinalPage;
+import org.jboss.tools.intellij.quarkus.utils.XPathDefinitions;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.List;
+
+import static com.intellij.remoterobot.search.locators.Locators.byXpath;
+
 /**
  * New project dialog fixture
  *
  * @author zcervink@redhat.com
  */
-@DefaultXpath(by = "MyDialog type", xpath = "//div[@class='DialogRootPane']")
+@DefaultXpath(by = "MyDialog type", xpath = XPathDefinitions.DIALOG_ROOT_PANE)
 @FixtureName(name = "New Project Dialog")
 public class QuarkusNewProjectFinalPage extends AbstractNewProjectFinalPage {
     public QuarkusNewProjectFinalPage(@NotNull RemoteRobot remoteRobot, @NotNull RemoteComponent remoteComponent) {
         super(remoteRobot, remoteComponent);
     }
+
+    @Override
+    public void setProjectName(String projectName) {
+        if (UITestRunner.getIdeaVersionInt() >= 20221) {
```

Comment:
This code should be in the test library please open an issue to track it

Reply:
@jeffmaury I have discussed this with @odockal today. **Do you know where the Quarkus wizard pages are implemented**? If it is a custom wizard implementation here in the repo (the XPaths and the UI structure looks that way), the code should definitely stay here. The only reason for moving that code to the library would be if the final page was the same as other final pages (I mean the implementation + XPaths - they differs form default final pages in IntelliJ).